### PR TITLE
bug: PopperArrow missing props as-child

### PR DESCRIPTION
**Pull request by D-Lite**
#353 

:) first time contributing here! Looking forward to your kind corrections😊

### CODE REVIEW

**Comment by onmax** [comment]
Legend!!

Thank you!

File: packages/radix-vue/src/shared/component/Arrow.vue [hunk 1]
```diff
@@ -39,6 +39,8 @@ const { primitiveElement } = usePrimitiveElement()
     :height="height"
     viewBox="0 0 30 10"
     preserveAspectRatio="none"
+    :as-child="props.asChild"
+    :as="as"
```
Comment (zernonia) [inline packages/radix-vue/src/shared/component/Arrow.vue:43]: let's remove this props assigment as we have `v-bind` to bind that props already 😁

File: packages/radix-vue/src/Popper/PopperArrow.vue [hunk 2]
```diff
@@ -67,6 +69,8 @@ const baseSide = computed(() =>
       :style="{
         display: 'block',
       }"
+      :as="as"
+      :as-child="props.asChild"
```
Comment (zernonia) [inline packages/radix-vue/src/Popper/PopperArrow.vue:73]: Nice.. this is the missing piece here! thanks for that @D-Lite ! Perhaps let replace `props.asChild` with `asChild`?

File: packages/radix-vue/src/shared/component/Arrow.vue (hunk 1 above)
Comment (D-Lite) [reply packages/radix-vue/src/shared/component/Arrow.vue:43]: Yeah, I forgot to clean up properly

File: packages/radix-vue/src/Popper/PopperArrow.vue (hunk 2 above)
Comment (D-Lite) [reply packages/radix-vue/src/Popper/PopperArrow.vue]: Great! made the fix and updated the pr

**Review by zernonia — APPROVED** [review]
lgtm!

**Pull request merged**